### PR TITLE
Microverse Review Activity Pull Request

### PR DESCRIPTION
**Pull request by Uzair-Manzoor**
modified for review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,8 +52,8 @@
             <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12 hero-text">
                 <p class="hero_desc">Hi,</p>
                 <p class="hero_desc">Welcome to My Portfolio Website!</p>
-                <p class="hero_desc I'm">I'm</p>
```
Comment (anyars-encarta) [inline index.html:58]: Great job with this project :muscle:
Could you kindly consider changing the class of I'm to something else? I will also suggest you avoid using apostrophes in class names.

File: style.css [hunk 2]
```diff
@@ -54,7 +54,7 @@ body, html {
     }
   
     50% {
-      transform: translateX(800px);
+      transform: translateX(-100px);
```
Comment (anyars-encarta) [inline style.css]: Wow! :+1:, Nice using the transform. I think using duplicate styling will affect the layout of your document. Coud you consider removing the duplicates?

File: style.css [hunk 3]
```diff
@@ -64,8 +64,9 @@ body, html {
   
   .imageAboutPage {
     animation-name: move;
-    animation-duration: 5s;
+    animation-duration: 2s;
```
Comment (anyars-encarta) [inline style.css]: Wow! 👍, Nice using the transform. I think using duplicate styling will affect the layout of your document. Could you consider removing the duplicates?

File: style.css [hunk 4]
```diff
@@ -1,160 +1,172 @@
...
+  box-shadow: 0 0 20px 0 rgba(184, 132, 132, 0.8);
 }
 
-body, html {
-    background: linear-gradient(rgb(0, 0, 0), rgb(39, 3, 3), rgb(0, 0, 0));
-    margin: 0;
-    background-size: cover;
-    position: relative;
-    background-repeat:repeat-y;
-    display:block;
```
Comment (HossainAraf) [inline style.css:13]: Display flex or grid is preferable.

**Review by anyars-encarta — CHANGES_REQUESTED** [review]
Great effort and nice css approach :heart_eyes:. I think you have to consider checking through your CSS and remove duplicate style declarations.

Happy Coding! :handshake:

**Pull request merged**